fishy/gui/splash.py

```diff
@@ -32,7 +32,7 @@ def show(self, win_loc, q):
 
         top.title("Loading...")
         top.resizable(False, False)
-        top.iconbitmap(helper.manifest_file('icon.ico'))
+        top.iconphoto(True, tk.PhotoImage(file='fishy/icon.png'))
```

Comment:
getting this on windows:

_tkinter.TclError: couldn't open "fishy/icon.png": no such file or directory

Reply:
Translation: The method of creating an icon in Linux is different, so I can solve the problem with a simple condition.